oci-seccomp-bpf-hook.go

```diff
@@ -126,11 +128,14 @@ func main() {
 		os.Exit(0)
 	}
 
-	profilePath, err := filepath.Abs(*fileName)
+	outputProfilePath, err := filepath.Abs(*outputFile)
 	if err != nil {
 		log.Error(err)
 	}
-
+	if !filepath.IsAbs(*inputFile) {
+		*inputFile = ""
+		logrus.Error("input filepath is not absolute")
```

Comment:
Let's treat the output and input file equally and force them to be absolute. This is something we should add to the documentation as well.

Also, we should error out (not only log) when the files are relative.

Reply:
Another thing we should do is to first check if they are actually specified:
```golang
if *inputFile != "" {

}
```

Reply:
Yes we should force absolute paths.  The input file should be optional.